From bd02ab3b3fe60cec12e260cddc54966ebfd75381 Mon Sep 17 00:00:00 2001
From: Yazan Shhady <yazan.shhady@solid-run.com>
Date: Tue, 20 Jun 2023 17:02:20 +0300
Subject: [PATCH] add u-boot reset suport for rzg2l-solidrun platforms

---
 board/solidrun/rzg2l/rzg2l-solidrun.c | 33 ++++++++++++++++++++++++++-
 1 file changed, 32 insertions(+), 1 deletion(-)

diff --git a/board/solidrun/rzg2l/rzg2l-solidrun.c b/board/solidrun/rzg2l/rzg2l-solidrun.c
index 1e7d7711e6..7a25927cde 100644
--- a/board/solidrun/rzg2l/rzg2l-solidrun.c
+++ b/board/solidrun/rzg2l/rzg2l-solidrun.c
@@ -62,6 +62,18 @@ DECLARE_GLOBAL_DATA_PTR;
 #define HcRhDescriptorA		0x048
 #define LPSTS			0x102
 
+/* WDT */
+#define WDT_BASE 					0x12800800
+#define WDTCNT						0x00
+#define WDTSET						0x04
+#define WDTTIM						0x08
+#define WDTINT						0x0C
+#define PECR						0x10
+#define PEEN						0x14
+#define WDTCNT_WDTEN					BIT(0)
+#define WDTINT_INTDISP					BIT(0)
+
+
 void s_init(void)
 {
 	/* SD1 power control: P39_1 = 0; P39_2 = 1; */
@@ -150,7 +162,26 @@ int board_init(void)
 	return 0;
 }
 
-void reset_cpu(void)
+static void wdt_write(u32 val, unsigned int reg)
 {
+	writel(val, WDT_BASE + reg);
+}
 
+static int reset_wdt_start()
+ {
+	/* Clear Lapsed Time Register and clear Interrupt */
+	wdt_write(WDTINT_INTDISP, WDTINT);
+	/* 2 consecutive overflow cycle needed to trigger reset */
+	wdt_write(0, WDTSET);
+	/* Initialize watchdog counter register */
+	wdt_write(0, WDTTIM);
+	/* Enable watchdog timer*/
+	wdt_write(WDTCNT_WDTEN, WDTCNT);
+ 
+	return 0;
+}
+
+void reset_cpu(void)
+{
+	reset_wdt_start();	
 }
-- 
2.25.1

