From b44449ac9f4d719118acb4977f78f409afd304b7 Mon Sep 17 00:00:00 2001
From: Mikhail Anikin <mikhail.anikin@solid-run.com>
Date: Wed, 16 Aug 2023 16:11:44 +0300
Subject: [PATCH 38/45] tlv: init new structure if the tlv_eeprom_read returned
 EINVAL

---
 cmd/tlv_eeprom.c | 14 +++++++++-----
 1 file changed, 9 insertions(+), 5 deletions(-)

diff --git a/cmd/tlv_eeprom.c b/cmd/tlv_eeprom.c
index d64d69f8..22a1a417 100644
--- a/cmd/tlv_eeprom.c
+++ b/cmd/tlv_eeprom.c
@@ -298,13 +298,17 @@ int do_tlv_eeprom(struct cmd_tbl *cmdtp, int flag, int argc, char *const argv[])
 		tlv = tlv_eeprom_read(tlv_eeprom_get_by_index(current_dev),
 								 0, eeprom, ARRAY_SIZE(eeprom));
 		if (IS_ERR(tlv)) {
-			printf("Failed to read EEPROM data from device: %ld\n",
-				   PTR_ERR(tlv));
-			tlv = NULL;
-			return 0;
+			if (tlv == -EINVAL) {
+				printf("Bad TLV data, initilising new EEPROM \n");
+				tlv = tlv_init(eeprom, ARRAY_SIZE(eeprom));
+			} else {
+				printf("Failed to read EEPROM data from device: %ld\n",
+					PTR_ERR(tlv));
+				tlv = NULL;
+			}
 		}
-
 		printf("EEPROM data loaded from device to memory.\n");
+		return 0;
 	}
 
 	// Subsequent commands require that the EEPROM has already been read.
-- 
2.41.0

