From fe456e56e348f81c0052e296adeaf151ca585652 Mon Sep 17 00:00:00 2001
From: Yazan Shhady <yazan.shhady@solid-run.com>
Date: Mon, 21 Aug 2023 12:21:28 +0300
Subject: [PATCH 2/2] add rzg2l-hummingboard-extended.dtb to renesas dtbs list

---
 arch/arm64/boot/dts/renesas/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/Makefile b/arch/arm64/boot/dts/renesas/Makefile
index f62c03e3cbc3..aef12b7039aa 100644
--- a/arch/arm64/boot/dts/renesas/Makefile
+++ b/arch/arm64/boot/dts/renesas/Makefile
@@ -95,6 +95,7 @@ dtb-$(CONFIG_ARCH_R9A07G044) += r9a07g044l2-smarc.dtb
 dtb-$(CONFIG_ARCH_R9A07G044) += r9a07g044l2-dev.dtb
 dtb-$(CONFIG_ARCH_R9A07G044) += rzg2l-hummingboard-ripple.dtb
 dtb-$(CONFIG_ARCH_R9A07G044) += rzg2l-hummingboard-tutus.dtb
+dtb-$(CONFIG_ARCH_R9A07G044) += rzg2l-hummingboard-extended.dtb
 
 dtb-$(CONFIG_ARCH_R9A07G054) += r9a07g054l2-smarc.dtb
 dtb-$(CONFIG_ARCH_R9A07G054) += r9a07g054l2-dev.dtb
-- 
2.25.1

